pkg/controller/nodeconfigpod/sync_configmaps.go

```diff
@@ -123,25 +123,31 @@ func (ncpc *Controller) syncConfigMaps(
 	ctx context.Context,
 	pod *corev1.Pod,
 	configMaps map[string]*corev1.ConfigMap,
-) error {
+) (error, bool) {
+	if !controllerhelpers.IsScyllaContainerDeployed(pod) {
```

Comment:
it feels like the `makeConfigMap` could handle transient states gracefully then making a precondition here

Reply:
also we get re-queued to pod changes to we don't need to manually setup timer for re-queuing